### PR TITLE
[TASK] Show firmware version in info box

### PR DESCRIPTION
**Pull request by mwarning**
The firmware should be shown in the info box.

### CODE REVIEW

**Comment by xf-** [comment]
Should be already returned by showFirmware.
https://github.com/ffrgb/meshviewer/pull/151/files#diff-1b7411720abb55d6bdfa6f6f09761b2cR37

**Comment by mwarning** [comment]
ah, it is not displayed for us because we do not have 'base'.

**Comment by xf-** [comment]
Yeah that should be changed

**Comment by mwarning** [comment]
@xf- fixed.

File: lib/infobox/node.js [hunk 1]
```diff
@@ -30,10 +30,14 @@ define(['sorttable', 'snabbdom', 'd3-interpolate', 'moment', 'helper'],
       var release = helper.dictGet(d.nodeinfo, ['software', 'firmware', 'release']);
       var base = helper.dictGet(d.nodeinfo, ['software', 'firmware', 'base']);
 
-      if (release === null || base === null) {
+      if (release === null) {
```
Comment (xf-) [inline lib/infobox/node.js]: should work in both cases.

```javascript
    function showFirmware(d) {
      return [
        helper.dictGet(d.nodeinfo, ['software', 'firmware', 'release']),
        helper.dictGet(d.nodeinfo, ['software', 'firmware', 'base'])
      ].filter(function (n) {
        return n !== null;
      }).join(' / ') || undefined;
    }
```

**Review by xf- — CHANGES_REQUESTED** [review]

File: lib/infobox/node.js (hunk 1 above)
Comment (mwarning) [reply lib/infobox/node.js]: @xf- sure, go ahead. :-)

**Comment by mwarning** [comment]
thanks

**Comment by xf-** [comment]
no problem - this should work in every case

**Review by xf- — APPROVED** [review]

**Pull request merged**